### PR TITLE
Prison pearl integration

### PR DESCRIPTION
**Pull request by TealNerd**
Add upgrading to prison pearls, summoning, returning, and separate fuel costs

### CODE REVIEW

**Comment by TealNerd** [comment]
@ProgrammerDan

**Comment by ProgrammerDan** [comment]
Looks solid. @CivFactions thoughts?

lifegoals: add tests

**Comment by TealNerd** [comment]
lol what is a test

File: pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	<groupId>com.devotedmc</groupId>
 	<artifactId>ExilePearl</artifactId>
 	<packaging>jar</packaging>
-	<version>1.1.2-SNAPSHOT</version>
+	<version>1.2.1-SNAPSHOT</version>
```
Comment (CivFactions) [inline pom.xml]: Rev to 2.0?

**Comment by CivFactions** [comment]
damn son, you did some serious work here. 

Only thing I can think to add is updating `resources/config.yml` with the new default values.

**Comment by ProgrammerDan** [comment]
@TealNerd gold star from both of us, make that resources/ config update and I'll merge post haste <3

**Comment by ProgrammerDan** [comment]
Now to testing, tweaking, and fleshing out the featurelist. Good baseline.

**Comment by TealNerd** [comment]
Yeah I've done a bit of testing but for some reason my test server crashed and I can't get into the control panel so... rip. Also thoughts on needed features?

**Comment by ProgrammerDan** [comment]
Depends on alignment to PP desired, but could include:

* interaction limitations (range from pearl, combat, block manip)
* check on EP restrictions, how do they filter against PP behavior (blocked from bastion entry, etc.)
* inv drop on ppsummon/ppreturn (was that in there? did I miss it?)
* flag to explicitly disable upgrades (does it do this gracefully if no upgrade items specified)
* admin command to upgrade a pearl to a PP, or downgrade from a PP to an EP
* ?? tired and forgetting what PP does any more, not sure which features are just abuse-vessels and which are valuable.
Pearl stealing? How is that handled currently -- (EP to PP, someone else pearls the PP'd person while summoned)

Fun stuff!

**Comment by TealNerd** [comment]
It should drop items when summoned, as for disabling upgrades, the recipe is set to bedrock when none is specified and it wont show up on the pearl the way it's written. The admin command to change pearl type now actually saves so that's possible as well. Not sure about pearl stealing though, forgot about that one.

**Pull request merged**